### DISCUSSION TITLE
installation problem

### DISCUSSION THREAD

**Discussion by Manuel-Baez-Ponce** [gerardog/gsudo#308] [Q&A] [answered]
Hello everyone.

I have installed the package from winget correctly but when I try to use it it returns the following error: "&: The term 'gsudo.exe' is not recognized as a name of a cmdlet, function, script file, or executable program. Check the spelling of
the name, or if a path was included, verify that the path is correct and try again." Do I have to do any prior configuration before I can use it?

I thank you in advance for the help you can provide me.

**Comment by gerardog** [comment, marked as answer]
Please restart your consoles after installing, to refresh the PATH environment variable.

**Comment by Manuel-Baez-Ponce** [comment]
Thanks, that has solved my problem.
